definitions/npm/ramda_v0.x.x/flow_v0.82.x-/ramda_v0.x.x.js

```diff
@@ -1354,8 +1354,8 @@ declare module ramda {
     src: { [k: string]: T }
   ): { [k: string]: T };
 
-  declare function evolve<A: Object>(NestedObject<Function>, A): A;
   declare function evolve<A: Object>(NestedObject<Function>): A => A;
+  declare function evolve<A: Object>(NestedObject<Function>, A): A;
```

Comment:
I'm curious if anyone has come across improved understanding as to why these kinds of things are necessary. To be clear: This is not to hold up the PR and instead for my (and perhaps others') edification.